galsim/base.py

```diff
@@ -2181,6 +2181,105 @@ def getScaleRadius(self):
         return self.SBProfile.getScaleRadius()
 
 
+class Spergel(GSObject):
+    """A class describing a Spergel profile.
+
+    The Spergel surface brightness profile is characterized by three properties: its Spergel index
+    `nu`, its `flux`, and either the `half_light_radius` or `scale_radius`.  Given these properties,
+    the surface brightness profile scales as I(r) ~ r^{nu} * K_{nu}(r), where K_{nu} is the
+    modified Bessel function of the second kind.
+
+    For more information, refer to
+
+        D. N. Spergel, "ANALYTICAL GALAXY PROFILES FOR PHOTOMETRIC AND LENSING ANALYSIS,"
+        ASTROPHYS J SUPPL S 191(1), 58-65 (2010) [doi:10.1088/0067-0049/191/1/58].
+
+    Initialization
+    --------------
+
+    The allowed range of values for the `nu` parameter is -0.85 <= nu <= 0.85.  An exception will be
+    thrown if you provide a value outside that range.  These values are chosen to match the values
+    investigated by the above paper.  At and below nu=0.0, the surface brightness profile diverges
+    at the origin, which currently affects our ability to perform photon-shooting or `real_space`
```

Comment:
The "which currently affects our ability..." is a bit vague.  It's not clear if this means you just can't do it at all, or can do it less accurately, or what.  I recommend being a little more specific here.